tests/e2e/cucumber/features/smoke/upload.feature

```diff
@@ -21,6 +21,7 @@ Feature: Upload
       | new-lorem-big.txt | txtFile | new lorem big file  |
       | lorem.txt         | txtFile | lorem file          |
       | textfile.txt      | txtFile | some random content |
+      | comma,.txt        | txtFile | comma               |
```

Comment:
@ScharfViktor Should we check for `comma,.txt` files? or they are not necessary?

Reply:
yeah, let's include this case. Could you put the issue link here, please?

Reply:
yeah, okey
Thanks